packages/botonic-react/src/util/dom.js

```diff
@@ -1,23 +1,26 @@
-export const getBotonicScrollableContent = () => {
-  return document.getElementById('botonic-scrollable-content')
-}
-
-export const getScrollableArea = () => {
+export const getScrollableArea = webchatElement => {
   const getArea = area => {
-    const botonicScrollableContent = getBotonicScrollableContent()
+    const botonicScrollableContent = webchatElement.querySelector(
+      '#botonic-scrollable-content'
+    )
     const scrollableArea =
-      botonicScrollableContent &&
-      botonicScrollableContent.getElementsByClassName(area)[0]
+      botonicScrollableContent && botonicScrollableContent.querySelector(area)
     return scrollableArea
   }
   return {
-    full: getArea('simplebar-content'),
-    visible: getArea('simplebar-content-wrapper'),
+    full: getArea('.simplebar-content'),
+    visible: getArea('.simplebar-content-wrapper'),
```

Comment:
should we add these references as constants? also `@botonic-scrollable-content`, etc.